tsconfig.base.json

```diff
@@ -437,6 +437,12 @@
       "@energinet-datahub/watt/validators": [
         "libs/ui-watt/src/lib/components/input/shared/validators/index.ts"
       ],
+      "@energinet-datahub/watt/watt-field": [
+        "libs/ui-watt/src/lib/components/field/index.ts"
+      ],
```

Comment:
consider not exposing this if we choose to use `watt-field` directly within `watt-text-field`.